deploy/common/k8sbased.sh

```diff
@@ -333,13 +333,23 @@ function launch_central {
         )
       fi
 
+      local helm_chart="$unzip_dir/chart"
+
       if [[ -n "$CI" ]]; then
-        helm lint "$unzip_dir/chart"
-        helm lint "$unzip_dir/chart" -n stackrox
-        helm lint "$unzip_dir/chart" -n stackrox "${helm_args[@]}"
+        helm lint "${helm_chart}"
+        helm lint "${helm_chart}" -n stackrox
+        helm lint "${helm_chart}" -n stackrox "${helm_args[@]}"
+      fi
+
+      if [[ -n "${CENTRAL_CHART_DIR_OVERRIDE}" ]]; then
+        helm_chart="${CENTRAL_CHART_DIR_OVERRIDE}"
       fi
```

Comment:
(**)

1. After looking at this code again, I don't see a reason why we shouldn't `helm lint` the override chart. If helm args changed between versions, we should find out about it. Compatibility tests seem appropriate for this kind of stuff. I suggest moving this block to go before the `helm lint ...` block.
2. Just realized now, sorry. It should be good to add a log message saying that we use override chart.

```suggestion
      if [[ -n "${CENTRAL_CHART_DIR_OVERRIDE}" ]]; then
        echo "Using override central helm chart from ${CENTRAL_CHART_DIR_OVERRIDE}"
        helm_chart="${CENTRAL_CHART_DIR_OVERRIDE}"
      fi
```

Reply:
So the reason I put it after `helm lint` previously was that `helm lint` did not accept the `--version` flag for some reason. Now that that is gone this may work, I'll try it.